Buildings/Applications/DHC/EnergyTransferStations/CoolingIndirect.mo

```diff
@@ -0,0 +1,316 @@
+    annotation (Line(points={{-69,0},{-20,0},{-20,48}}, color={0,0,127}));
+  connect(TBuiRet.port_a, hex.port_b2) annotation (Line(points={{-70,-60},{0,-60},
+          {0,-6},{20,-6}}, color={0,127,255}));
+  connect(pro.y, cp.u)
+    annotation (Line(points={{1,110},{18,110}}, color={0,0,127}));
+  connect(senMasFlo.m_flow, pro.u2)
+    annotation (Line(points={{-50,71},{-50,104},{-22,104}}, color={0,0,127}));
+  connect(senTDisSup.T, dTDis.u1)
+    annotation (Line(points={{-80,71},{-80,122},{-62,122}}, color={0,0,127}));
+  connect(senTDisRet.T, dTDis.u2) annotation (Line(points={{80,71},{80,80},{-70,
+          80},{-70,110},{-62,110}}, color={0,0,127}));
+  connect(dTDis.y, pro.u1)
+    annotation (Line(points={{-39,116},{-22,116}}, color={0,0,127}));
+  connect(cp.y, int.u)
+    annotation (Line(points={{41,110},{58,110}}, color={0,0,127}));
+  connect(int.y, Q)
+    annotation (Line(points={{81,110},{110,110}}, color={0,0,127}));
+  connect(Q_flow, cp.y) annotation (Line(points={{110,150},{50,150},{50,110},{41,
+          110}}, color={0,0,127}));
+  annotation (defaultComponentName="coo",
+    Icon(coordinateSystem(preserveAspectRatio=false), graphics={
+        Rectangle(
+          extent={{-100,-56},{100,-64}},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          pattern=LinePattern.None),
+        Rectangle(
+          extent={{-100,64},{100,56}},
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          pattern=LinePattern.None),
+        Rectangle(
+          extent={{-80,80},{80,-80}},
+          lineColor={175,175,175},
+          fillColor={35,138,255},
+          fillPattern=FillPattern.Solid),
+        Text(
+          extent={{-52,40},{54,-40}},
+          lineColor={0,0,0},
+          fillColor={35,138,255},
+          fillPattern=FillPattern.Solid,
+          textStyle={TextStyle.Bold},
+          textString="ETS")}),                                   Diagram(
+        coordinateSystem(preserveAspectRatio=false, extent={{-100,-100},{100,160}})),
+    Documentation(info="<html>
+<p>
+Indirect cooling energy transfer station (ETS) model that controls the building chilled water supply temperature 
+by modulating a primary control valve on the district supply side. The design is based on a typical district 
+cooling ETS described in ASHRAE's 
+<a href=\"https://www.ashrae.org/technical-resources/bookstore/district-heating-and-cooling-guides\">
+District Cooling Guide</a>.  
+As shown in the figure below, the building pumping design (constant, variable) is specified on the building
+side, not within the ETS. 
+</p>
+<p align=\"center\">
+<img src=\"modelica://Buildings/Resources/Images/Applications/DHC/EnergyTransferStations/CoolingIndirect.png\" alt=\"DHC.ETS.CoolingIndirect\"/>
+</p>
+<h4>Reference</h4>
+<p>
+American Society of Heating, Refrigeration and Air-Conditioning Engineers. (2013). Chapter 5: End User Interface. In <i>District Cooling Guide</i>. 1st Edition. 
```

Comment:
change line to make it shorter, so that when reviewing it, it's more convenient and don't need to slide the horizon bar to see the contents.

Reply:
Good catch. Thanks, will be done.